src/pages/resultsView/enrichments/ExpressionEnrichmentsContainer.tsx

```diff
@@ -32,11 +44,18 @@ export interface IExpressionEnrichmentContainerProps {
         nameOfEnrichmentDirection?: string,
         count: number,
         color?: string
-    }[]
+    }[];
+    sampleKeyToSample: {
+        [uniqueSampleKey: string]: Sample;
+    }
+    groupSampleAxisData: IStringAxisData
```

Comment:
I find this name confusing. what does this data represent?

Reply:
It represents groups(horizontal) axis data

Reply:
you mean which group each sample is in?

Reply:
yes

Reply:
how about `sampleGroupMembershipData: IStringAxisData`?